### DISCUSSION TITLE
"This setting is managed by your administrator" - Help !!

### DISCUSSION THREAD

**Discussion by mohit83k** [undergroundwires/privacy.sexy#214] [General]
2 years ago i bought my PC and used privacy.sexy to manage privacy on my windows OS. After using it, there was this message in various places including in setting that "This [feature] is managed by your administrator". 
![managedbyadmin](https://user-images.githubusercontent.com/17878921/229297546-33964aec-0478-4cb3-a685-a8e5305f15eb.PNG)

It is causing problem for me and i want to revert it back but unable to do so. Please help me to revert it back. 

Issue is that when i change the setting, it keep reverting back. 

I'm asking on this forum because it was the cause. 
Thanks

**Comment by ItsIgnacioPortal** [comment]
Did you search "managed" in privacy.sexy? 

You can change those settings using a GPO if you have Windows Enterprise.

Are you absolutely certain that it was privacy.sexy who did this? Because in my experience its Win10Privacy that does that. You shouldn't trust proprietary software
